app/controllers/comments_controller.rb

```diff
@@ -1,23 +1,17 @@
 class CommentsController < ApplicationController
 
   def create
-    @comment = Comment.new comment_params
+    @comment = Comment.new comment_params  
     @comment.account_id = current_account.id
-
-    respond_to do |format|
-      format.js {
-        if @comment.save
-          @comments = Comment.where(post_id: @comment.post_id)
-          render "comments/create"
-        else
-          # unable to save
-        end
-      }
-    end
+    if @comment.save
+      redirect_to community_post_path(@comment.post_id, @comment.post), flash: {success: "comment was created successfully!" }
+    else
+      redirect_to community_post_path(@comment.post_id, @comment.post), flash: { danger: "comment was not saved!" }
```

Comment:
this seems to have a duplicate of the redirect_to, can you make it DRY?

Reply:
Yes Magesh